fernet.py

```diff
@@ -19,18 +23,13 @@ class InvalidSignature(Exception):
 
 
 class Fernet:
-    """Pure python Ferent module
-    see https://github.com/fernet/spec/blob/master/Spec.md
-    """
```

Comment:
Sorry, I've been running a local copy lately and I removed the link. I'll put it back.